course-definition.yml

```diff
@@ -1486,7 +1486,7 @@ stages:
     name: "Multi Replica Command Propagation"
     difficulty: hard
     description_md: |
-      In this stage, you'll extend your implementation to support propagating commands to multiple replicas.
+      In this stage, you'll extend your implementation of the master to support propagating commands to multiple replicas. No need to change anything in the replica implementation.
```

Comment:
@andy1li let's keep the intro limited to a single short sentence - maybe we could move the extra info to notes?

Reply:
Might be slightly outdated, but here's the last doc we have on how to structure stage instructions: https://docs.codecrafters.io/contributors/authoring-challenges/writing-stage-instructions